index.html

```diff
@@ -81,146 +81,263 @@
         maxTocLevel: 4,
         inlineCSS: true
       };
-
-      var includeTerms = [ "identity profile", "entity", "entity credential", "subject", "claim" ];
     </script>
   </head>
   <body>
     <section id='abstract'>
       <p>
-A self-sovereign architecture for verifiable claims is one where the
-holder of a verifiable claim is in complete control of their
-identifier, where their verifiable claims are stored, and how they are
-used. There is currently no widely used self-sovereign,
-privacy-enhancing standard for expressing and transacting verifiable
-claims (aka: credentials, attestations) via the Web.
+We typically use claims made by others about us to gain access to
+various online services. For example, we use
+email addresses to identify ourselves to online services,
+driver's licenses to prove that we are capable of operating a
+motor vehicle, university degrees to prove we are well-trained
+and knowledgeable, and government-issued passports to travel
+between countries and access financial services. It is the
```

Comment:
The lead in sentence for this section indicated that we use claims to gain access to online services, yet, this following sentence that is meant to enumerate examples of that -- actually enumerates examples of non-online services and regulatory compliance. We should change the lead in:

"We use claims made by others about us to gain access to services or to prove our compliance with rules and regulations."

But that last bit is too formal for this introduction. I'll have to try and think of a more colloquial way of saying that.